numpy/ma/tests/test_core.py

```diff
@@ -447,6 +447,16 @@ def test_deepcopy(self):
         assert_equal(copied.mask, [0, 0, 0])
         assert_equal(a.mask, [0, 1, 0])
 
+    def test_format(self):
+        # Note: Python 3.6 and above fstring uses __format__, see PEP 498.
+        # Testing with underlying primitive.
```

Comment:
Fyi, `"{: >5}".format(a)` also uses `format`, f-strings are not a new mechanism.

Reply:
I think we can just drop the whole comment, I do not think it really adds anything.

Reply:
Agreed, I think dropping this comment is sensible